test/e2eTests/pages/windows/deletePatientModal.js

```diff
@@ -0,0 +1,74 @@
+const Modal = require('./modal');
+
+const LOG_TAG = '[DeletePatientModal]';
+
+class DeletePatientModal extends Modal {
```

Comment:
This feels more like a component. Models are part of the page just like other components, so not making this a component feels odd. It removes it from the existing page-component architecture, which might make explaining and maintaining the architecture more difficult. I'm about to write documentation and this feels a bit out of place."

What do you think?

Reply:
I started with that approach but felt the modal wasn't part of a page but rather was an independent object...some sort of "page" that is part of a main page. Weird! Let's see how angular defines it.